src/js/components/AuthorSelector.js

```
@@ -3,232 +3,232 @@
+	CardBody,
+	CardFooter,
+	Icon,
+	Notice,
+	Flex,
+	FlexItem,
+	Dashicon,
+	Tooltip
 } from '@wordpress/components';
 import { useState, useEffect } from '@wordpress/element';
 import { search, people, plusCircle, info } from '@wordpress/icons';
 import { sprintf } from '@wordpress/i18n';
```

ESLint

[error] '@wordpress/i18n' import is duplicated.

(no-duplicate-imports)